doc/src/manual/methods.md

```diff
@@ -263,10 +263,7 @@ Possible fix, define
   g(::Float64, ::Float64)
 ```
 
-Here the call `g(2.0, 3.0)` could be handled by either the `g(Float64, Any)` or the `g(Any, Float64)`
-method, and neither is more specific than the other. In such cases, Julia raises a [`MethodError`](@ref)
-rather than arbitrarily picking a method. You can avoid method ambiguities by specifying an appropriate
-method for the intersection case:
+The order in which the methods are executed does not matter and neither is more specific than the other.  Here the call `g(2.0, 3.0)` could be handled by either the `g(Float64, Any)` or the `g(Any, Float64)` method. In such cases, Julia raises a [`MethodError`](@ref) rather than arbitrarily picking a method. You can avoid method ambiguities by specifying an appropriate method for the intersection case:
```

Comment:
I would change this from methods are executed to methods are defined.

Reply:
Agreed. I've made the suggested edit.